src/main/java/Duke/exceptions/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package Duke.exceptions;
+
+public class DukeException extends Exception {
+    public DukeException(String errorMessage) {
```

Comment:
Can consider adding an empty line here so as to standardize with the rest of the files, just something trivial 👍